apps/react-workshop/src/Tooltip.stories.tsx

```diff
@@ -3,91 +3,55 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import React from 'react';
-import { Tooltip } from '@itwin/itwinui-react';
+import { Button, Tooltip } from '@itwin/itwinui-react';
 export default {
   title: 'Tooltip',
 };
 
 export const Top = () => {
   return (
     <Tooltip placement='top' content='Here I am!'>
-      <div
-        id='tooltip-target'
-        style={{
-          marginTop: 40,
-          marginLeft: 100,
-          width: 'fit-content',
-        }}
-      >
-        Please, try to hover me!
-      </div>
+      <Button id='tooltip-target'>Please, try to hover me!</Button>
     </Tooltip>
   );
 };
 
 export const Right = () => {
   return (
     <Tooltip placement='right' content='Here I am!'>
-      <div
-        id='tooltip-target'
-        style={{
-          marginTop: 40,
-          marginLeft: 100,
-          width: 'fit-content',
-        }}
-      >
-        Please, try to hover me!
-      </div>
+      <Button id='tooltip-target'>Please, try to hover me!</Button>
     </Tooltip>
   );
 };
 
 export const Bottom = () => {
   return (
     <Tooltip placement='bottom' content='Here I am!'>
-      <div
-        id='tooltip-target'
-        style={{
-          marginTop: 40,
-          marginLeft: 100,
-          width: 'fit-content',
-        }}
-      >
-        Please, try to hover me!
-      </div>
+      <Button id='tooltip-target'>Please, try to hover me!</Button>
     </Tooltip>
   );
 };
 
 export const Left = () => {
   return (
     <Tooltip placement='left' content='Here I am!'>
-      <div
+      <Button
         id='tooltip-target'
         style={{
-          marginTop: 40,
           marginLeft: 100,
```

Comment:
Why does this `Button` still have some of the inline styles?

Reply:
I left the inline style for `marginLeft` because without it, the tooltip shows up on the right since there isn't enough space on the left.

Reply:
But I just realized that the leftover `width` style was not necessary and so removed it (https://github.com/iTwin/iTwinUI/pull/2066/commits/0cb85e23d9e3342de264000b11e9452db8797961).